artifacts/deploy/bootstrap-token-configuration.yaml

```diff
@@ -149,13 +147,12 @@ rules:
   - namespaces
   verbs:
   - get
-  - list
-  - watch
-  - create
 - apiGroups:
   - ""
   resources:
   - secrets
+  # resourceNames:
+  # - {{clustername}}-impersonator
   verbs:
   - get
   - list
```

Comment:
Please confirm whether `list` and `watch` permissions for the `Secrets` resource are needed

Reply:
I've checked the code and tested in my local environment, they can be removed and will not cause any error.

Reply:
Removed.